scripts/add-cahps-measures.js

```diff
@@ -0,0 +1,91 @@
+// Libraries
+const fs    = require('fs');
+const parse = require('csv-parse');
+const path  = require('path');
+
+const additionalMeasuresFilepath = '../util/additional-measures.json'
+var additionalMeasures = require(additionalMeasuresFilepath);
+
+// We want to overwrite CAHPS measures every time we run this script.
```

Comment:
we should do this in the `parse` callback, since that's where we're inserting the measures, and it's quick to understand if these are adjacent